ddtrace/_trace/tracer.py

```diff
@@ -416,17 +417,17 @@ def get_log_correlation_context(self) -> Dict[str, str]:
         if self.enabled or self._apm_opt_out:
             active = self.context_provider.active()
 
+        trace_id = 0
+        span_id = 0
         if isinstance(active, Span) and active.service:
             service = active.service
         else:
             service = config.service
         if active:
```

Comment:
We don't need to check if the span exists. This is handled by `_get_trace_details_for_log_injection`
```suggestion
```